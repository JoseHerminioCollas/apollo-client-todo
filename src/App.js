import React from 'react'
import './app.scss'

import Todos from './component/Todo'
import AddTodo from './component/AddTodo'

const App = () => (
  <section className="todo-app">
    <header>
      Todos
    </header>
    <AddTodo />
    <Todos />
  </section>
)

export default App
