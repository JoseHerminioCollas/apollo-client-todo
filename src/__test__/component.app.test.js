import React from 'react'
import renderer from 'react-test-renderer'

import App from '../App'

it('matches a former snapshot', () => {
  const component = renderer.create(<App />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
