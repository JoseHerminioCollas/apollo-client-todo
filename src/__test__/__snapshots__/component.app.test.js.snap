// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo Application matches a former snapshot 1`] = `
<section
  className="todo-app"
>
  <header>
    Apollo Todos
  </header>
  <section
    className="add-todo"
  >
    <header>
      Add A Todo
    </header>
    <form>
      <label>
        <span>
          Title
        </span>
        <input
          name="title"
        />
      </label>
      <label>
        <span>
          Description
        </span>
        <input
          name="description"
        />
      </label>
      <button
        onClick={[Function]}
        type="submit"
      >
        Add Todo
      </button>
    </form>
  </section>
  <div>
    Loading...
  </div>
</section>
`;
